src/Parts/Embed.php

```diff
@@ -22,7 +22,7 @@ class Embed
     public ?EmbedProvider $provider;
     public ?EmbedAuthor $author;
     /**
-     * @var \Ragnarok\Fenrir\Parts\EmbedField[]
+     * @var EmbedField[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)